tests/test_plugins/test_invdes.py

```diff
@@ -140,6 +140,13 @@ def post_process_fn(sim_data: td.SimulationData, **kwargs) -> float:
     return anp.sum(intensity.values)
 
 
+postprocess_obj = tdi.Sum(
+    operations=[
```

Comment:
Wondering whether `terms` would be more intuitive instead of `operations`?

Reply:
the classes are also called `Operations` so it kind of makes sense. But yea I wonder if there's an even better name for these?

Reply:
Yeah don't know, `terms` only really makes sense in the context of `Sum` probably. Maybe it's fine as-is..